Update: Licensing Admin API updates and Troubleshooting section links (#7174)
Kong/docs.konghq.com: app/_includes/md/enterprise/deploy-license.md

For more detail and options, see the
[Admin API `licenses` endpoint reference](/gateway/latest/licenses/examples).

> It is recommended to [restart](https://docs.konghq.com/gateway/latest/production/running-kong/systemd/#restart-kong-gateway) the Kong Gateway nodes after applying or updating a license.

Reviewer: ```suggestion
We recommend [restarting](/gateway/{{page.release}}/reference/cli/#kong-restart) the {{site.base_gateway}} nodes after applying or updating a license.
```
This might be a better link. The systemd page is a bit too specific.
